storage/Utils/SystemCmd.cc

```diff
@@ -88,14 +88,23 @@ namespace storage
     void
     SystemCmd::init()
     {
+        _childStdin = NULL;
 	_files[0] = _files[1] = NULL;
-	_pfds[0].events = _pfds[1].events = POLLIN;
+	_pfds[0].events = POLLOUT; // stdin
+	_pfds[1].events = POLLIN;  // stdout
+	_pfds[2].events = POLLIN;  // stderr
```

Comment:
No, that just enables getting notifications when data is available for any of them. The file descriptors still remain separate.